tls/s2n_x509_validator.h

```diff
@@ -131,6 +131,12 @@ s2n_cert_validation_code s2n_x509_validator_validate_cert_chain(struct s2n_x509_
 s2n_cert_validation_code s2n_x509_validator_validate_cert_stapled_ocsp_response(struct s2n_x509_validator *validator,  struct s2n_connection *conn,
                                                                                 const uint8_t *ocsp_response, uint32_t size);
 
+/**
+ * Checks that the peer's certificate chain has been received and validated.
```

Comment:
Without receiving the certificate chain s2n-tls cannot validate the cert chain, it seems repetitive to say received: 

```suggestion
 * Checks whether the peer's certificate chain has been validated.
```

Reply:
I think it's worth explicitly calling out "received" here, since even if a user doesn't care whether or not the certificate has been validated, the certificate hasn't been received / isn't available at all until after validation.